run-bungeecord.sh

```diff
@@ -43,11 +33,13 @@ case "${TYPE^^}" in
   ;;
 
   CUSTOM)
-    if isURL ${BUNGEE_JAR_URL}; then
-      BUNGEE_JAR=$BUNGEE_HOME/$(basename ${BUNGEE_JAR_URL})
-    elif [[ -f ${BUNGEE_JAR_URL} ]]; then
+    if [[ -v BUNGEE_JAR_URL ]]; then
       echo "Using custom server jar at ${BUNGEE_JAR_URL} ..."
-      BUNGEE_JAR=${BUNGEE_JAR_URL}
+      BUNGEE_JAR=$BUNGEE_HOME/$(basename ${BUNGEE_JAR_URL})
+      download_required=true
```

Comment:
It looks like `download_required=true` needs to be initialized as the default towards the top of the script since with all default settings, the startup fails. Here's with bash execution debugging enabled:

```
docker run -it --rm --entrypoint bash bungee
bash-4.4# bash -x /usr/bin/run-bungeecord.sh
+ : BUNGEECORD
+ : 512m
+ : 1.0.0
+ BUNGEE_HOME=/server
+ RCON_JAR_URL=https://github.com/orblazer/bungee-rcon/releases/download/v1.0.0/bungee-rcon-1.0.0.jar
+ echo 'Resolving type given BUNGEECORD'
Resolving type given BUNGEECORD
+ case "${TYPE^^}" in
+ : https://ci.md-5.net/job/BungeeCord
+ : lastStableBuild
+ : https://ci.md-5.net/job/BungeeCord/lastStableBuild/artifact/bootstrap/target/BungeeCord.jar
+ : lastStableBuild
+ BUNGEE_JAR=/server/BungeeCord-lastStableBuild.jar
+ '[' '' = true ']'
+ '[' -d /plugins ']'
+ [[ -n '' ]]
+ '[' TRUE = TRUE ']'
+ [[ ! -e /server/plugins/bungee-rcon-1.0.0.jar ]]
+ echo 'Downloading rcon plugin'
Downloading rcon plugin
+ mkdir -p /server/plugins/bungee-rcon
+ curl -sSL -o /server/plugins/bungee-rcon-1.0.0.jar https://github.com/orblazer/bungee-rcon/releases/download/v1.0.0/bungee-rcon-1.0.0.jar
+ echo 'Copy rcon configuration'
Copy rcon configuration
+ sed -i 's#${PORT}#25575#g' /tmp/rcon-config.yml
+ sed -i 's#${PASSWORD}##g' /tmp/rcon-config.yml
+ mv /tmp/rcon-config.yml /server/plugins/bungee-rcon/config.yml
+ rm -f /tmp/rcon-config.yml
+ '[' -d /config ']'
+ '[' -f /var/run/default-config.yml -a '!' -f /server/config.yml ']'
+ '[' '' = TRUE ']'
+ '[' 0 == 0 ']'
+ chown -R bungeecord:bungeecord /server
+ echo 'Setting initial memory to 512m and max to 512m'
Setting initial memory to 512m and max to 512m
+ JVM_OPTS='-Xms512m -Xmx512m '
+ '[' 0 == 0 ']'
+ exec sudo -E -u bungeecord java -Xms512m -Xmx512m -jar /server/BungeeCord-lastStableBuild.jar
Error: Unable to access jarfile /server/BungeeCord-lastStableBuild.jar
```